app/main.py

```diff
@@ -0,0 +1,19 @@
+import logging
+from fastapi import FastAPI
+from app.endpoints.pdf_processing import router as pdf_processing_router
+from app.endpoints.questions import router as question_router
+
+# Configurar el logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    handlers=[
+        logging.FileHandler("app.log"),  # Guarda los logs en el archivo app.log
```

Comment:
` logging.FileHandler("app.log")`,I like this but just as a comment: When using containers this is not recommended. The file will increase in size and when the container is restarted the info will be deleted.